pyccl/halos/halo_model_base.py

```diff
@@ -202,6 +203,7 @@ def _get_bsigma(self, cosmo, sigM, a):
             float or array_like: f(sigma_M) function.
         """
 
+    @mass_def_api
```

Comment:
Do we need this also in the deprecated `get_halo_bias`?

Reply:
True - actually, we *only* need it there! Good call.

Reply:
done